mountpoint-s3/tests/fuse_tests/mod.rs

```diff
@@ -84,7 +87,8 @@ mod mock_session {
     impl TestClient for MockTestClient {
         fn put_object(&mut self, key: &str, value: &[u8]) -> Result<(), Box<dyn std::error::Error>> {
             let full_key = format!("{}{}", self.prefix, key);
-            self.client.add_object(&full_key, value.into());
```

Comment:
Would it be better to do this in the `From` implementation for MockObject? (mock_client.rs:161)